src/customize-pop/customize-desktop.md

```diff
@@ -0,0 +1,99 @@
+# Customizing the Desktop
+
+Change the function of the `SUPER` key, desktop backgrounds, workspace features, and placement and visibility of desktop components in `Settings` ➞ `Desktop`.
+
+- [`Desktop Options`](/customize-pop/customize-desktop.md#desktop-options) provides options to modify the `SUPER` key's function, enabling Hot Corner, and adding or positioning top bar elements.
+- [`Background` settings](/customize-pop/customize-desktop.md#background) allows users to set pre-installed wallpaper or user images as the background image.
+- [`Appearance` settings](/customize-pop/customize-desktop.md#appearance) provides the option to choose a dark or light theme that applies across the entire desktop environment.
+- [`Dock` settings](/customize-pop/customize-desktop.md#dock) provides option to modify Dock visibility, placement, size, and displayed icons.
+- [`Workspaces` settings](/customize-pop/customize-desktop.md#workspaces) includes options configure workspace positioning and behavior.
+
+---
+
+## Desktop Options
+
+Customize basic desktop elements in the `Desktop Options` tab.
+
+![General Settings](/images/customize-desktop/general-settings.png)
+
+| Setting | Function |
+|----------|----------|
+| Super Key Action | Assigns the `SUPER` key to start the Launcher, display Workspace navigation, or view installed applications. |
+| Hot Corner | Displays workspace navigation when the user moves the mouse cursor to the upper left corner of the screen. |
+| Top Bar | Display or hide items along the top bar of the main home screen. |
+| Window Controls | Enable or disable the minimize and maximize buttons in system and application windows. |
+
+## Background
+
+Choose a pre-installed desktop background or assign a custom image in the `Background` tab. You can also right-click anywhere on the desktop and select `Change Background...`.
+
+>**Note**: Background adjustments such as centered, scaled, spanned, stretched, and zoom can be applied using the GNOME Tweaks application: [GNOME Tweaks Appearance Settings](gnome-tweaks-extensions.md#appearance)
```

Comment:
Consider `Background modes ... can be set` instead of `Background adjustments ... can be applied`; a list of adjustments sounds like a list of things that can be individually applied (not mutually exclusive.)

Reply:
I'm unresolving this for further discussion on two points:

1. The main point of my suggestion was to change the word `adjustments` to `modes`; changing `apply` to `set` was just an artifact of that. If you want to keep the word `adjustments` specifically, that's fine, but I would not use `set` with `adjustments`. Either `modes can be set` like I suggested or `adjustments can be applied` like you had originally.
2. It's now sticking out to me that `zoom` sounds like an oddball following `centered, scaled, spanned, stretched`; I would expect to read `zoomed`. You could also add `tiled` since that is what the (ambiguously named) `Wallpaper` option in GNOME Tweaks does.

Reply:
Those are all good points. I thought I had implemented your change, but I'm still seeing the original text. I never intentionally resolve comments without implementing the suggested change. If it's something I disagree with, I'll make a comment and leave the issue unresolved.